packages/closer/components/SubscriptionCards/index.tsx

```
@@ -76,88 +97,109 @@ const SubscriptionCards = ({
+                    </Heading>
+                    <ul className="mb-4">
+                      {plan.perks.split(',').map((perk) => {
+                        return (
+                          <li
+                            key={perk}
+                            className="bg-[length:16px_16px] bg-[center_left] bg-[url(/images/subscriptions/bullet.svg)] bg-no-repeat pl-6 mb-1.5"
+                          >
+                            <span className="block">
+                              {perk.includes('<') ? (
+                                <span
+                                  dangerouslySetInnerHTML={{ __html: perk }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)